### PR TITLE
visual tweaking of Dashboard page

### PR DESCRIPTION
**Pull request by screaser**
An attempt to try to bring the current shell more in-line with the
design… there may be better ways to do this (I used explicit CSS in
layout.scss)… if so please teach me!

The background image is not currently working. Have not yet been able
to debug why so for now I reverted main.html to just use grey instead
of the background texture.

### CODE REVIEW

File: web/client/templates/layout.html [hunk 1]
```diff
@@ -29,11 +29,15 @@
 
 <template name="userNav">
   <nav>
-    <div class="nav-wrapper green lighten-1">
-      <a href="#" class="brand-logo center">AmDash</a>
+    <div class="nav-wrapper grey darken-3">
       <ul class="left">
-        <li><a data-action="settings"><i class="material-icons">settings</i></a></li>
+        <li><i class="material-icons medium pad-left">equalizer</i></li>
+        <li><a href="#" class="brand-logo pad-left">AmDash</a></li>
```
Comment (bzwheeler) [inline web/client/templates/layout.html]: Materialize seems to do something on medium screens (tablet size) with the header.  The 'AmDash' title centers and overlays the 'Your Dashboard' text.

File: web/client/templates/overview/overview.html [hunk 2]
```diff
@@ -18,23 +18,23 @@
 </template>
 
 <template name="nextPayment">
-  <h2 class="card-title center grey-text lighten-2">next payment</h2>
-  <div class="card-panel white">
-    <h3>in {{daysToNextPayment}} days</h3>
+  <div class="dash-card-title"><span class="dash-card-title-text">next payment</span></div>
```
Comment (bzwheeler) [inline web/client/templates/overview/overview.html]: If text size was the reason you switched away from the h tags, you can customize that with the '$h[1-6]-fontsize' sass variables.  Semantically, the h2 tags are the right thing to use here but not a huge deal.

**Comment by bzwheeler** [comment]
Visually this is looking really good, nice work.  For the font-size, I'd recommend setting the materialize sass variables for h2 rather than using a custom class name to override it.  That way all h2's look consistent without having to remember a class and later if we need a differently styled h2 we can at that point add a modifying class.

To do that, I'd recommend adding a `variables.scss` file in client/stylesheets with `$h2-fontsize: 1.6rem;'`and then update `main.scss` to import `variables.scss` right after the default variable import `@import "materialize/components/variables";` on line 9

**Comment by screaser** [comment]
Ah — that helps thanks! 

I was puzzling over how best to do that kind of thing without actually modifying _variables.scss — which would I assume get overwritten later if we update materialize.

If you are in the mood to do it don’t let me hold you back… otherwise I’ll get to it tomorrow/Wed. Either way but I don’t get offended if someone wants to tweak my code.

-me

--
Steve Reaser // 919.673.3009 // @screaser
SproutwiseKids.com // ItsAlwaysMorning.com
--

**Comment by bzwheeler** [comment]
Cool, I'll be working on stuff in the morning so I'll go ahead and make that change and get this PR merged.

**Pull request merged**